mvnw

```diff
@@ -0,0 +1,236 @@
+#!/bin/sh
```

Comment:
Why do we need mvwrapper? See related discussion: https://github.com/airlift/airbase/pull/97